src/p_enemy.c

```diff
@@ -1035,14 +1043,20 @@ void A_Look(mobj_t *actor)
 {
   mobj_t *targ;
 
+  targ = actor->subsector->sector->soundtarget;
```

Comment:
Can this be const?

Reply:
It is later used in `P_SetTarget` and `P_CheckSight`, so we need to change the parameter declarations for these functions.

Reply:
Then it's okay.